### DISCUSSION TITLE
Is it possible to target a specific version with the new package installer?

### DISCUSSION THREAD

**Discussion by t0roide** [inventree/InvenTree#3811] [Q&A] [answered]
Hello again!
The title is pretty self explanatory, just wondering if with the new option for bare-metal installation version targeting is possible.

Many thanks in advance

**Comment by matmair** [comment, marked as answer]
Hi there @t0roide !
You can select the version in your package manager
```bash
sudo apt install inventree=0.8.0
```

I am currently working on an interactive installer that also asks for version numbers etc. - not ETA on that as I am working on a lot of things at the same time.

**Reply by t0roide** [reply]
Great! Thank you so much for the swift response @matmair !

**Reply by matmair** [reply]
A lot of the enterprise features are currently developed by me because that is my area of interest. If you want feel free to jump in, we are happy to guide new ppl. through the code base.

**Reply by t0roide** [reply]
As of now, I don't feel like I have the time or the confidence to get into it. But at some point in the near future I would love to contribute to inventree, I just can't at the moment.

**Reply by matmair** [reply]
No pressure, was meant as an offer.
